src/main/java/tech/ada/extends_insights/domain/models/requests/UserRequest.java

```diff
@@ -0,0 +1,20 @@
+package tech.ada.extends_insights.domain.models.requests;
+
+import lombok.Getter;
+import tech.ada.extends_insights.domain.entities.User;
+
+import java.time.LocalDateTime;
+
+@Getter
+public class UserRequest {
+    private Long userId;
```

Comment:
O UserRequest é usado para o usuário fazer uma requisição quando for fazer um POST para criar um usuário novo, então não precisamos desse userId pois ele vai ser criado automático e não vai ser passado pelo body na requisição.